src/components/OutputForm.tsx

```diff
@@ -144,5 +145,12 @@ export const OutputTime = ({ label, time, relative }: { label: string; time?: nu
     };
   });
 
-  return <OutputField label={label}>{time && <RelativeTimeText time={time} baseTime={nowTime} defaultToTime={!relative} />}</OutputField>;
+  let useRelative = relative ?? false;
+  if (relative == undefined && time) {
+    // If the time is within 1 day of today, use relative time.
+    const dateDiff = differenceInCalendarDays(new Date(), new Date(time));
+    if (Math.abs(dateDiff) <= 1) useRelative = true;
```

Comment:
This doesn't feel like the right place to do this. Maybe `RelativeTimeText` should have a prop like relativeDaysRange that applies this logic so that individual implementations can just pass an integer.

`<RelativeTimeText time={time} baseTime={nowTime} relative={useRelative} relativeDaysRange=1 />`